CONTRIBUTING.md

```diff
@@ -9,11 +9,15 @@ and ask that all contributors do the same.
 
 ### Git Protocol
 
-To contribute to this project, people internal to 18F can create a branch and submit a pull request. If you are external to 18F, you can fork the repository and submit a pull request that way.
+To contribute to this project, people internal to 18F can create a branch and 
+submit a pull request. If you are external to 18F, you can fork the repository 
+and submit a pull request that way.
 
-We are minimizing commits on the `master` branch by squashing commits and force pushing them to `master`. 
+We are minimizing commits on the `master` branch by squashing commits and force 
+pushing them to `master`. 
 
-For a more detailed walk through on how to do this, you can read thoughtbot's [Git Protocol](https://github.com/thoughtbot/guides/tree/master/protocol/git#write-a-feature) document.
+For a more detailed walk through on how to do this, you can read thoughtbot's [Git Protocol](https://github.com/thoughtbot/guides/tree/master/protocol/git#write-a-feature) 
```

Comment:
hey, wanna rebase against master? there are some changes in this file that I just merged in.